### PR TITLE
Start v1.8.1.1 release candidate

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

**Comment by gadenbuie** [comment]
I know this change is small, but I would like to advocate for us sticking with semantic versioning and calling this **v1.8.2** instead of **v1.8.1.1**. In general, I think it's best to avoid these non-standard version numbers except for the (hopefully very) rare case where we need to both make an out-of-cycle hotfix and have also already moved the version number past the most logical next choice.

**Comment by cpsievert** [comment]
> I think it's best to avoid these non-standard version numbers

Do you have reasoning for this beyond sticking to semantic versioning? This is a practice we've done for quite a while and I do like that the version itself indicates it's a patch release

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "homepage": "https://shiny.rstudio.com",
   "repository": "github:rstudio/shiny",
   "name": "@types/rstudio-shiny",
-  "version": "1.8.1-alpha.9000",
+  "version": "1.8.1-alpha.1",
```
Comment (cpsievert) [inline package.json:6]:
```suggestion
  "version": "1.8.1-1",
```

**Comment by gadenbuie** [comment]
> This is a practice we've done for quite a while and I do like that the version itself indicates it's a patch release

I mean, the version format is `<major>.<minor>.<patch>` so in that regard it's not really a patch release – at least not in the semantic versioning format. semver.org outlines includes pre-release and build information, but those are separated from the version number by `-<pre-release>` and `+<build>`.

I think that it's one thing to step out of semver because we're forced to push a hotfix to CRAN that we hadn't accounted for, but we should adhere to semver as much as possible. In this case, nothing is forcing us to add a non-standard version component.

**Comment by cpsievert** [comment]
I'll let @wch have final say on the version (we had both agreed that `1.8.1.1` was more appropriate)

**Pull request merged**